examples/jsm/loaders/GLTFLoader.js

```diff
@@ -2315,7 +2315,7 @@ class GLTFParser {
 
 		// Use an ImageBitmapLoader if imageBitmaps are supported. Moves much of the
 		// expensive work of uploading a texture to the GPU off the main thread.
-		if ( typeof createImageBitmap !== 'undefined' && /^((?!chrome|android).)*safari/i.test( navigator.userAgent ) === false ) {
+		if ( typeof createImageBitmap !== 'undefined' && /^((?!chrome|android).)*safari/i.test( navigator.userAgent ) === false && navigator.userAgent.match( /Firefox\/([0-9]+)\./ )?.[ 1 ] < 98 === false ) {
```

Comment:
Could we do this without `?.` optional chaining?

Reply:
I'll rewrite the check without optional chaining tomorrow 👍 . 

It's indeed more safe to avoid the syntax for now.

Reply:
I can understand that we want to write the version detection code short but `navigator.userAgent.match( /Firefox\/([0-9]+)\./ )?.[ 1 ] < 98 === false` might a little bit tricky especially for first readers? They may think why it isn't `navigator.userAgent.match( /Firefox\/([0-9]+)\./ )?.[ 1 ] >= 98`.

I know this detection code works fine because `navigator.userAgent.match( /Firefox\/([0-9]+)\./ )?.[ 1 ]` will be `undefined` on non-Firefox and `undefined < 98` will be `false`. But it took time a bit for me to understand.

(Sorry if my overthinking.)

And not a big preference, but personally I like adding a comment about likely when we can remove the version detection code https://github.com/mrdoob/three.js/issues/23906#issuecomment-1101405508

By the way I didn't know that numbers and strings can be compared in JavaScript.

Reply:
What do you think about the new version of the PR? Tried to make the check easier to read.

Reply:
Much more straightforward, thanks.

Nit:

```
const isFirefox = navigator.userAgent.toLowerCase().indexOf( 'firefox' ) > - 1;
const firefoxVersion = isFirefox ? navigator.userAgent.match( /Firefox\/([0-9]+)\./ )[ 1 ] : - 1;
```
The first line uses `toLowerCase()` while the second line doesn't. I think it would be better to make them consistent unless there is any special reason.

Reply:
Good catch! Updated!